sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaSquidSensor.java

```diff
@@ -56,7 +63,6 @@
   private final DefaultJavaResourceLocator javaResourceLocator;
   private final Settings settings;
   private final NoSonarFilter noSonarFilter;
-  private final PostAnalysisIssueFilter postAnalysisIssueFilter;
 
   public JavaSquidSensor(SonarComponents sonarComponents, FileSystem fs,
     DefaultJavaResourceLocator javaResourceLocator, Settings settings, NoSonarFilter noSonarFilter, PostAnalysisIssueFilter postAnalysisIssueFilter) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "postAnalysisIssueFilter". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)